.gitignore

```diff
@@ -24,9 +26,13 @@ etc/dev-frontend.yml
 .env.production.local
 go.work
 go.work.sum
+/zrok-venv
```

Comment:
why an absolute path here?

Reply:
I had put this in here to have a place for the sdk's virtual environment for development. It isn't strictly necessary, but seemed like a nice to have for me.

Reply:
I don't think `/zrok-venv` is inside of the git repo though? If it were just `zrok-venv`, it would make sense to me.

Reply:
I did intend it to be in the root, but can for sure drop the / and have it be anywhere. Or just remove it entirely

Reply:
I'd just drop the `/` myself.